### PR TITLE
add rad-validate tool and how do i article

### PR DESCRIPTION
**Pull request by elmiko**
This change adds a script for validating that the resource.yaml has been
installed properly. It also contains an article describing how to use
the script.

### CODE REVIEW

**Comment by ruivieira** [comment]
@elmiko lGtm.

Perhaps macOS could be added to https://github.com/radanalyticsio/radanalyticsio.github.io/pull/231/files#diff-5714f32a8cd4683a1dd34c6bb6339cd5R11 since `rad-validate` works fine on a Mac terminal, out of the box.

File: assets/tools/rad-validate [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+echo "* scanning for radanalytics.io objects"
+rad_valid=yes
+missing_objects=""
+
+check_component()
+{
+    if oc get "$1" "$2" &> /dev/null; then
+        if [[ "$rad_valid" == "yes" ]]; then
+            rad_valid=yes
+        fi
```
Comment (Jiri-Kremser) [inline assets/tools/rad-validate]: hmm, these 3 lines are not necessary, are they?

File: assets/tools/rad-validate [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+echo "* locating oc command"
+if ocloc=$(which oc) &> /dev/null; then
```
Comment (Jiri-Kremser) [inline assets/tools/rad-validate]: nit: some distros don't have the `which` command present, what about using `type` as a fallback?
something like

`if ocloc=$(which oc || type -p oc) &> /dev/null; then`

File: assets/tools/rad-validate (hunk 1 above)
Comment (elmiko) [reply assets/tools/rad-validate]: mostly leftover from testing, i can remove them

File: assets/tools/rad-validate (hunk 2 above)
Comment (elmiko) [reply assets/tools/rad-validate]: thanks! i will change this to use `command`

**Comment by ruivieira** [comment]
@elmiko lgtm

**Pull request merged**